modules/link-in-bio/assets/scss/frontend.scss

```diff
@@ -1,3 +1,5 @@
+// TODO: Remove this file in v3.27.0 [ED-15717]
```

Comment:
Why can't we remove it now?

Reply:
@louiswol94 We need to keep it for BC purposes, for users who have updated Core but who haven't updated Pro. 

E.g. Pro = 3.24 and Core 3.26. > Pro will expect the frontend.scss file.